src/NHibernate.TestDatabaseSetup/NHibernate.TestDatabaseSetup.csproj

```diff
@@ -7,19 +7,24 @@
     <IsTestProject>true</IsTestProject>
     <NoWarn>$(NoWarn);3001;3002;3003;3005</NoWarn>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+  <PropertyGroup Condition="$(NhNetCoreApp)">
     <OutputType>Exe</OutputType>
     <GenerateProgramFile>false</GenerateProgramFile>
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\NHibernate.Test\NHibernate.Test.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="NUnit" Version="3.12.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
+    <PackageReference Include="NUnit" Version="3.13.2" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
-    <PackageReference Include="NUnitLite" Version="3.12.0" />
+  <ItemGroup Condition="$(NhNetFx)">
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
```

Comment:
Let's use same version as in master.

Reply:
4.1.0? Why not, although I do not see the point of not upgrading this dependency to 4.2.1 here then in master when merging 5.3.x.